unlock-app/src/services/storageService.js

```diff
@@ -80,10 +80,11 @@ export class StorageService extends EventEmitter {
    * TODO: consider a more robust url building
    * @param {*} senderAddress
    */
-  async getTransactionsHashesSentBy(senderAddress) {
+  async getRecentTransactionsHashesSentBy(senderAddress) {
```

Comment:
May be worth including the default time interval and reason for change in the JSDoc above to indicate why this is different from the old one